projects/valtimo/components/src/lib/components/carbon-list/carbon-list.component.html

```diff
@@ -118,7 +118,7 @@
 </ng-template>
 
 <ng-template #rowDisabled let-data="data">
-  <div *ngIf="data.locked" class="disabled">
+  <div *ngIf="data.locked" class="disabled locked">
```

Comment:
Classes disabled and locked I think are meant to accomplish the same behaviour, so remove disabled